src/js/components/help/downloads.js

```diff
@@ -0,0 +1,324 @@
+    isPublic: false,
+    canAccess: ({ isEnterprise, tenantCapabilities }) => isEnterprise || tenantCapabilities.canDelta
+  },
+  {
+    id: 'mender-cli',
+    title: 'Mender CLI',
+    getLocations: defaultLocationFormatter,
+    canAccess: () => true,
+    osList: ['darwin', 'linux']
+  },
+  {
+    id: 'mender-configure-module',
+    packageId: 'mender-configure',
+    title: 'Mender configure',
+    getLocations: multiArchLocationFormatter,
+    canAccess: ({ tenantCapabilities }) => tenantCapabilities.hasDeviceConfig,
+    osList: defaultOSVersions,
+    archList: ['all']
+  },
+  {
+    id: 'mender-connect',
+    title: 'Mender connect',
+    getLocations: multiArchLocationFormatter,
+    canAccess: () => true,
+    osList: defaultOSVersions,
+    archList: defaultArchitectures
+  },
+  {
+    id: 'mender-convert',
+    title: 'Mender convert',
+    getLocations: ({ versionInfo }) => [
+      {
+        location: `https://github.com/mendersoftware/mender-convert/archive/refs/tags/${getVersion(versionInfo, 'mender-convert')}.zip`,
+        title: 'Mender convert'
+      }
+    ],
+    canAccess: ({ tenantCapabilities }) => tenantCapabilities.hasDeviceConnect
+  },
+  {
+    id: 'mender-gateway',
+    title: 'Mender gateway',
+    getLocations: multiArchLocationFormatter,
+    isPublic: false,
+    canAccess: ({ isEnterprise }) => isEnterprise,
+    osList: defaultOSVersions,
+    archList: defaultArchitectures
+  },
+  {
+    id: 'monitor-client',
+    packageId: 'mender-monitor',
+    title: 'Monitor-client',
+    getLocations: multiArchLocationFormatter,
+    isPublic: false,
+    canAccess: ({ tenantCapabilities }) => tenantCapabilities.hasMonitor,
+    osList: defaultOSVersions,
+    archList: ['all']
+  }
+];
+
+const copyOptions = [
+  { id: 'curl', title: 'Curl command', format: defaultCurlDownload },
+  { id: 'gitlab', title: 'Gitlab Job definition', format: defaultGitlabJob }
+];
+
+const DocsLink = ({ className = '', docsVersion, path, title }) => (
+  <a className={className} href={`https://docs.mender.io/${docsVersion}${path}`} target="_blank" rel="noopener noreferrer">
+    {title} <Launch style={{ verticalAlign: 'text-bottom' }} fontSize="small" />
+  </a>
+);
+
+const DownloadSection = ({ docsVersion, item, isEnterprise, versionInformation, os, setSnackbar, tokens }) => {
+  const [anchorEl, setAnchorEl] = useState();
+  const [open, setOpen] = useState(false);
+  const [currentLocation, setCurrentLocation] = useState('');
+  const { id, getLocations, packageId, title } = item;
+  const locations = getLocations({ isEnterprise, tool: item, versionInfo: versionInformation.repos, os });
+
+  const handleToggle = event => {
+    const anchor = anchorEl ? null : event?.currentTarget.parentElement;
+    setAnchorEl(anchor);
+    setCurrentLocation(event?.target.getAttribute('value'));
+  };
+
+  const handleSelection = useCallback(
+    ({ target: { value } }) => {
+      const option = copyOptions[value];
```

Comment:
I don't know why, but it always copies the curl command. I could never get the gitlab job...